### PR TITLE
provide python2 operator model namespace setup

### PR DESCRIPTION
**Pull request by sdickes**
To support using python 2.7 provide a com.ibm.streamsx.topology.functional.python2 namespace for application topology operators. 
toolkit/build.xml was modified to copy files from com.ibm.streamsx.topology.functional.python
modified topology.py to use the appropriate namespace -- based on python version -- when adding operators to the graph

### CODE REVIEW

**Comment by ddebrunner** [comment]
Can one of the admins verify this patch?

File: toolkit/build.xml [hunk 1]
```diff
@@ -28,6 +28,15 @@
    <mkdir dir="${tk}/license"/>
    <copy file="${streamsx.topology}/LICENSE" todir="${tk}/license"/>
 
+   <copy todir="${tk}/com.ibm.streamsx.topology.functional.python2">
+     <fileset dir="${tk}/com.ibm.streamsx.topology.functional.python"
```
Comment (ddebrunner) [inline toolkit/build.xml]: Why three filesets, won't this result in copying everything?

Doesn't excludes support multiple items so a single fileset can be used?

File: toolkit/build.xml [hunk 2]
```diff
@@ -51,10 +60,15 @@
     <delete>
        <fileset dir="${tk}">
           <include name="com.ibm.streamsx.topology.functional.java/**"/>
+          <include name="com.ibm.streamsx.topology.functional.python2/**"/>
           <include name="com.ibm.streamsx.topology.file/**"/>
           <include name="com.ibm.streamsx.topology.testing/**"/>
+          <exclude name="com.ibm.streamsx.topology.functional.python2/pyversion.sh"/>
           <exclude name="**/namespace-info.spl"/>
        </fileset>
+       <dirset dir="${tk}">
```
Comment (ddebrunner) [inline toolkit/build.xml]: Is this needed given the include at line 63?

**Comment by ddebrunner** [comment]
test now

**Comment by ddebrunner** [comment]
test now

File: toolkit/build.xml (hunk 2 above)
Comment (sdickes) [inline toolkit/build.xml]: I found that the PyFunction* directories would not be removed otherwise.

File: toolkit/build.xml (hunk 1 above)
Comment (sdickes) [inline toolkit/build.xml]: corrected

**Comment by ddebrunner** [comment]
test now

**Comment by ddebrunner** [comment]
Can one of the admins verify this patch?

**Comment by ddebrunner** [comment]
build failure was just me checking out a new build setup.

**Comment by ddebrunner** [comment]
test now

**Comment by sdickes** [comment]
Did an automatic build fail?

**Comment by ddebrunner** [comment]
Test com.ibm.streamsx.topology.test.splpy.PythonFunctionalOperatorsTest failed

**Comment by ddebrunner** [comment]
Seems like it might be not specific to this pr, let me investigate.

**Comment by ddebrunner** [comment]
test now

**Pull request merged**